tests/basic-tests.js

```diff
@@ -108,8 +108,8 @@ describe("Basic Tests", () => {
       const pathSuffixes = ["0/0", "0/1", "100/100"];
       const ui = await flowMultiPrompt(this.speculos, [
         [{header:"Sign",body:"Transaction"}],
-        [{header:"Transfer",body:"12345 to denali12yp9cc0melq83a5nxnurf0nd6fk4t224dtg0lx"}],
-        [{header:"Fee",body:"123444444"}],
+        [{header:"Transfer",body:"0.000012345 to denali12yp9cc0melq83a5nxnurf0nd6fk4t224dtg0lx"}],
+        [{header:"Fee",body:"0.123444444"}],
```

Comment:
I'm thinking that a couple more tests should be added to better test the various edge cases,  e.g.:

1.  One that requires the maximum number of digits
2.  One that doesn't print a decimal point
3.  Maybe one for each of the different number of digits shown after the decimal point?
4.  Others?